ontology/ngbo.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: ontology_detail
+id: ngbo
+title: Next Generation Biobanking Ontology
+contact:
+  email: dal.alghamdi92@gmail.com
+  github: dalalghamdi
+  label: Dalia Alghamdi
+  orcid: 0000-0002-2801-0767
+description: 'The Next Generation Biobanking Ontology (NGBO) is an open application ontology representing contextual data about omics digital assets in biobank. The ontology focuses on capturing the information about three main activities: wet bench analysis used to generate omics data, bioinformatics analysis used to analyze and interpret data, and data management. The scope of the ontology includes real-world physical objects such as specimens, computerized objects such as datasets, and planned processes and related specifications. The intended end-users are biobanking organizations, data collectors, omics researchers, and decision-makers. 
+
+I followed the Open Biological and Biomedical Ontologies (OBO) Foundry principles, and used basic formal ontology (BFO) as a top-level ontology for NGBO development. After careful assessment, we reused pre-existing terms from other appropriately maintained ontologies, annotated with proper PURLs, labels, and textual and logical definitions, For example, we directly imported terms from OBI that describe entities involved in biomedical investigations, such as, but not limited to, material entities, processes, and protocols, through OWL import. Additionally, from the ontology for biobanking (OBIB; https://github.com/biobanking/biobanking), we imported terms to describe specimens, donors, and specimen management [62]. Information artifact ontology (IAO; https://github.com/information-artifact-ontology/IAO) is another relevant ontology used as a mid-level ontology and includes terms that describe identifiers, software solutions, and documents. 
```

Comment:
Best not to use personal pronouns in the description of the resource. Also, you can better take advantage of markdown formatting for links. There also seems to be a reference that is not substantiated (what is `[62]` supposed to point to?)